stable/yourls/values.yaml

```
@@ -0,0 +1,285 @@
+
+## Site of the application
+## Used to construct short URLs
+## ref: https://hub.docker.com/_/yourls/
+##
+yourlsSite: http://yourls
+
+## User of the application
+## ref: https://hub.docker.com/_/yourls/
+##
+yourlsUsername: user
+
+## Application password
+## Defaults to a random 10-character alphanumeric string if not set
+## ref: https://hub.docker.com/_/yourls/
+##
+# yourlsPassword:
+
+## Table prefix
+## ref: https://hub.docker.com/_/yourls/
+##
+yourlsTablePrefix: yourls_
+
+## Set to `false` to allow the container to be started with blank passwords
+## ref: https://hub.docker.com/_/yourls/
+allowEmptyPassword: true
+
+# Extra environment variables which will be saved in a release-specific secret
+# or retrieved via valueFrom.
+# extraEnv:
+# - name: SECRET_TO_SAVE
+#   value: secret_value
+# - name: AWS_ACCESS_KEY_ID
+#   valueFrom:
+#     secretKeyRef:
+#       name: existing-secret
+#       key: access-key-id
+extraEnv: []
+
+replicaCount: 1
+
+externalDatabase:
+## All of these values are only used when mariadb.enabled is set to false
+  ## Database host
+  host: localhost:3306
+
+  ## non-root Username for YOURLS Database
+  user: yourls
+
+  ## Database password
+  password: ""
+
+  ## Database name
+  database: yourls
+
+##
+## MariaDB chart configuration
+##
+## https://github.com/helm/charts/blob/master/stable/mariadb/values.yaml
+##
+mariadb:
+  ## Whether to deploy a mariadb server to satisfy the applications database requirements. To use an external database set this to false and configure the externalDatabase parameters
+  enabled: true
+  ## Disable MariaDB replication
+  replication:
+    enabled: false
+
+  ## Create a database and a database user
+  ## ref: https://github.com/bitnami/bitnami-docker-mariadb/blob/master/README.md#creating-a-database-user-on-first-run
+  ##
+  db:
+    name: yourls
+    user: yourls
+    ## If the password is not specified, mariadb will generates a random password
+    ##
+    # password:
+
+  ## MariaDB admin password
+  ## ref: https://github.com/bitnami/bitnami-docker-mariadb/blob/master/README.md#setting-the-root-password-on-first-run
+  ##
+  # rootUser:
+  #   password:
+
+  ## Enable persistence using Persistent Volume Claims
+  ## ref: http://kubernetes.io/docs/user-guide/persistent-volumes/
+  ##
+  master:
+    persistence:
+      enabled: true
+      ## mariadb data Persistent Volume Storage Class
+      ## If defined, storageClassName: <storageClass>
+      ## If set to "-", storageClassName: "", which disables dynamic provisioning
+      ## If undefined (the default) or set to null, no storageClassName spec is
+      ##   set, choosing the default provisioner.  (gp2 on AWS, standard on
+      ##   GKE, AWS & OpenStack)
+      ##
+      # storageClass: "-"
+      accessMode: ReadWriteOnce
+      size: 8Gi
+
+## Kubernetes configuration
+## For minikube, set this to NodePort, elsewhere use LoadBalancer or ClusterIP
+##
+service:
+  type: LoadBalancer
```

Oh and also, this parameter is used but not documented :wink: 
```suggestion
  type: LoadBalancer
  
  ## When using web.service.type: clusterIP, sets the user-specified cluster IP
  # clusterIP: ""
```